templates/repo/view_file.tmpl

```diff
@@ -111,6 +111,15 @@
 						{{end}}
 					</tbody>
 				</table>
+				<div class="code-view-menu-list ui fluid popup transition hidden">
+					<div class="ui column relaxed equal height">
+						<div class="column">
+							<div class="ui link list">
+								<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{URLJoin AppUrl AppSubUrl .RepoLink}}/src/commit/{{.CommitID}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.issues.context.reference_issue"}}</a>
```

Comment:
Don't just AppUrl to AppSubUrl - AppSubUrl is already in AppUrl - (I appreciate it's confusing)

```suggestion
								<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{URLJoin AppUrl .RepoLink}}/src/commit/{{.CommitID}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.issues.context.reference_issue"}}</a>
```